### PR TITLE
[Assignment1] Datos de dataset rellenos

### PR DESCRIPTION
**Pull request by Luxoimum**
He rellenado la linea con los datos de mi dataset.

### CODE REVIEW

File: Assignment1/DatasetDescriptions.csv [hunk 1]
```diff
@@ -1 +1,2 @@
-Your name; Your GitHub user; Dataset name; Dataset URL; Dataset brief description
+Your name; Your GitHub user; Dataset name; Dataset URL; Dataset brief description;
+Federico Pereyra L�pez;https://github.com/Luxoimum;Bibliotecas P�blicas de Madrid;https://datos.gob.es/es/catalogo/l01280796-bibliotecas-publicas-de-madrid-catalogo;;Ofrece informaci�n sobre los fondos de todas las bibliotecas p�blicas que han suscrito un convenio con la Comunidad de Madrid, y a los cuales cualquier ciudadano puede acceder con un carn� �nico de forma gratuita.
```
[the remark this answers is not in the document]
Comment (Luxoimum) [reply Assignment1/DatasetDescriptions.csv]: i re-encoded the file, hope it works properly!

**Comment by Luxoimum** [comment]
i've solved conflicts with utf8 and made Assignment2, hope everything was fine.

File: Assignment2/FedericoPereyraLopez.ttl [hunk 2]
```diff
@@ -0,0 +1,9 @@
+:Class01	includes :Sensor029;
+			includes :Computer101.
+
+:Sensor029	hasMeasurement	:Measurement8401.
+:Computer101	hasOwner	:User10A.
+:User10A	hasName	"Pedro".
+
+:Measurement8401	hasTemperature	"29";
```
Comment (ocorcho) [inline Assignment2/FedericoPereyraLopez.ttl]: LAcking datatypes here and in the next one.

File: Assignment2/FedericoPereyraLopez.ttl [hunk 3]
```diff
@@ -0,0 +1,9 @@
+:Class01	includes :Sensor029;
```
Comment (ocorcho) [inline Assignment2/FedericoPereyraLopez.ttl]: LAcking prefixes for all properties and for the base prefix.

**Comment by ocorcho** [comment]
Please check the ttl file, which still has pending comments.

**Comment by ocorcho** [comment]
Closing this pull request, since there has been no reaction

**Pull request closed without merging**